templates/galaxy/config/job_conf.yml

```diff
@@ -371,28 +373,19 @@ galaxy_jobconf:
             shell: /bin/bash
             resolve_dependencies: "true"
             identifier: "/data/0/cvmfs/singularity/python:2.7.12--1"
+
+      # TPV conda singularity destination
     - id: condor_singularity_with_conda_python2
       runner: condor
-      env:
-        GALAXY_MEMORY_MB: "{MEMORY_MB}"
-        GALAXY_SLOTS: "{PARALLELISATION}"
-        LC_ALL: C
       params:
-        priority: -{PRIORITY}
-        request_cpus: "{PARALLELISATION}"
-        request_memory: "{MEMORY}"
-        requirements: 'GalaxyGroup == "compute"'
-        tmp_dir: "True"
-        metadata_strategy: "extended"
-        require_container: true
         singularity_enabled: true
         # can and should be made stricter at some point
-        singularity_volumes: "$_CONDOR_SCRATCH_DIR:rw,$job_directory:rw,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/data/db/:ro,/data/dnb01/galaxy_db/:ro,/data/dnb02/galaxy_db/:ro,/data/dnb03/galaxy_db/:ro,/data/dnb05/galaxy_db/:ro,/data/dnb06/galaxy_db/:rw,/data/dnb07/galaxy_db/:rw,/data/dp01/galaxy_db/:rw,/data/0/galaxy_db/:ro,/data/1/galaxy_db/:ro,/data/2/galaxy_db/:ro,/data/3/galaxy_db/:ro,/data/4/galaxy_db/:ro,/data/5/galaxy_import/galaxy_user_data/:ro,/data/6/galaxy_db/:ro,/data/7/galaxy_db/:ro,/usr/local/tools/:ro"
+        singularity_volumes: "$_CONDOR_SCRATCH_DIR:rw,$job_directory:rw,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw" #/data/db/:ro,/data/dnb01/galaxy_db/:ro,/data/dnb02/galaxy_db/:ro,/data/dnb03/galaxy_db/:ro,/data/dnb05/galaxy_db/:ro,/data/dnb06/galaxy_db/:rw,/data/dnb07/galaxy_db/:rw,/data/dp01/galaxy_db/:rw,/data/0/galaxy_db/:ro,/data/1/galaxy_db/:ro,/data/2/galaxy_db/:ro,/data/3/galaxy_db/:ro,/data/4/galaxy_db/:ro,/data/5/galaxy_import/galaxy_user_data/:ro,/data/6/galaxy_db/:ro,/data/7/galaxy_db/:ro,/usr/local/tools/:ro"
```

Comment:
/usr/local/tools is where conda and so on lives ... if this is not part of the container, it can not use the conda envs I think. 

And the other volumes, what if a very old file in one of these volumes should be accessed? those are also no not in the container, correct?